packages/plans-grid-next/src/components/plan-button/style.scss

```diff
@@ -1,62 +1,75 @@
 @import "@automattic/onboarding/styles/mixins";
 
-.plan-features-2023-grid__actions-button {
+.button.plan-features-2023-grid__actions-button {
```

Comment:
This is causing a regression in the start with free link modal:

![Screenshot(191)](https://github.com/user-attachments/assets/61efb10e-fda9-4615-8d59-558c052d2d27)

Reply:
Fixed by 11862ce7f919c0cd7634fb3d4e5d550ce82caaf9